model/EvhrToA.py

```diff
@@ -540,7 +544,8 @@ def _orthoOne(bandFile, orthoDir, demDir, outSrsProj4, mapproject_threads,
 
             except RuntimeError as e:
 
-                raise RuntimeError(str(e) + ' Band file: ' + str(bandFile))
+                raise RuntimeError(str(e) + ' Band file: ' + str(bandFile)) \
```

Comment:
I did not know about the "from" style of raising exceptions. It seems to provide clearer messaging when the errors are raised.